ui/apps/platform/src/Containers/Compliance/Entity/ComplianceEntityCluster.jsx

```diff
@@ -1,14 +1,13 @@
 import React, { useContext, useState } from 'react';
 
-import entityTypes from 'constants/entityTypes';
 import EntityCompliance from 'Containers/Compliance/widgets/EntityCompliance';
 import ResourceCount from 'Containers/Compliance/widgets/ResourceCount';
 import ClusterVersion from 'Containers/Compliance/widgets/ClusterVersion';
 import Query from 'Components/CacheFirstQuery';
 import usePermissions from 'hooks/usePermissions';
 import { CLUSTER_NAME as QUERY } from 'queries/cluster';
 // TODO: this exception will be unnecessary once Compliance pages are re-structured like Config Management
-/* eslint-disable-next-line import/no-cycle */
+/* eslint-disable import/no-cycle */
```

Comment:
Curious about the `import/no-cycle`. Seems like it's because of the `SidePanel` component. Any thoughts on the effort for resolving the dependency cycle? I know sometimes a cycle is _acceptable_ if it's well understood. Do you feel like that's the case here?